.github/scripts/push-docker.sh

```diff
@@ -2,28 +2,48 @@
 
 set -e
 
-TAG=$1
-USER=$2
-PASSWORD=$3
-GHCR_USER=$4
-GHCR_PASSWORD=$5
+TAG="$1"
+USER="$2"
+PASSWORD="$3"
+GHCR_USER="$4"
+GHCR_PASSWORD="$5"
 
 rm -rf ~/.docker/config.json
 mkdir -p ~/.docker
 echo "{ \"experimental\": \"enabled\" }" > ~/.docker/config.json
 
 docker login -u "$USER" -p "$PASSWORD"
 
-docker manifest create savonet/liquidsoap:$TAG --amend savonet/liquidsoap-ci-build:${TAG}_amd64 --amend savonet/liquidsoap-ci-build:${TAG}_arm64 savonet/liquidsoap-ci-build:${TAG}_armhf
-docker manifest push savonet/liquidsoap:$TAG
+docker manifest create \
+  "savonet/liquidsoap:$TAG" \
+  --amend "savonet/liquidsoap-ci-build:${TAG}_amd64" \
+  --amend "savonet/liquidsoap-ci-build:${TAG}_arm64" \
+  "savonet/liquidsoap-ci-build:${TAG}_armhf"
 
-docker manifest create savonet/liquidsoap-alpine:$TAG --amend savonet/liquidsoap-ci-build:${TAG}_alpine_amd64 --amend savonet/liquidsoap-ci-build:${TAG}_alpine_arm64 savonet/liquidsoap-ci-build:${TAG}_alpine_armhf
-docker manifest push savonet/liquidsoap-alpine:$TAG
+docker manifest push "savonet/liquidsoap:$TAG"
+
+docker manifest create \
+  "savonet/liquidsoap-alpine:$TAG" \
+  --amend "savonet/liquidsoap-ci-build:${TAG}_alpine_amd64" \
+  --amend "savonet/liquidsoap-ci-build:${TAG}_alpine_arm64" \
+  "savonet/liquidsoap-ci-build:${TAG}_alpine_armhf"
+
+docker manifest push "savonet/liquidsoap-alpine:$TAG"
 
 docker login ghcr.io -u "$GHCR_USER" -p "$GHCR_PASSWORD"
 
-docker manifest create ghcr.io/savonet/liquidsoap:$TAG --amend ghcr.io/savonet/liquidsoap-ci-build:${TAG}_amd64 --amend ghcr.io/savonet/liquidsoap-ci-build:${TAG}_arm64 ghcr.io/savonet/liquidsoap-ci-build:${TAG}_armhf
-docker manifest push ghcr.io/savonet/liquidsoap:$TAG
+docker manifest create \
+  "ghcr.io/savonet/liquidsoap:$TAG" \
+  --amend "ghcr.io/savonet/liquidsoap-ci-build:${TAG}_amd64" \
+  --amend "ghcr.io/savonet/liquidsoap-ci-build:${TAG}_arm64" \
+  "ghcr.io/savonet/liquidsoap-ci-build:${TAG}_armhf"
+
+docker manifest push "ghcr.io/savonet/liquidsoap:$TAG"
+
+docker manifest create \
+  "ghcr.io/savonet/liquidsoap-alpine:$TAG" \
+  --amend "ghcr.io/savonet/liquidsoap-ci-build:${TAG}_alpine_amd64" \
+  --amend "ghcr.io/savonet/liquidsoap-ci-build:${TAG}_alpine_arm64" \
+  --amend "ghcr.io/savonet/liquidsoap-ci-build:${TAG}_alpine_armhf"
```

Comment:
Unsure if there is some kind of inconsistency. The last manifest create command has 3 --amend while others don't.

Reply:
Looks like it works both ways but let's make it consistent.